chapters/annotations.tex

```diff
@@ -72,6 +73,20 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 <a href="modelica:/MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
 
+\subsubsection{Style Sheet}
+
+A top-level package may inside the \lstinline!Documentation! have an annotation \lstinline!styleSheet="modelica:/Package/Resources/style.css"!, which provides a cascading style sheet.
+It may also be a vector of strings that corresponding to different style sheets that will be cascaded in the given order.
+The given style sheet(s) is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations of the package.
+\begin{nonnormative}
+It is recommended to use \lstinline[language=CSS]!class! and \lstinline[language=CSS]!id! selectors with a \lstinline[language=CSS]!$NameOfLibrary$-! prefix to avoid collisions when the content is included in a larger context.
+\end{nonnormative}
+The provided style sheet should not use type or universal selectors, due to possible interference with tool-specific styling.
+
+Vendors shall use a \lstinline[language=CSS]!$NameOfVendor$-! prefix to style vendor generated HTML content surrounding the user provided documentation.
```

Comment:
If we say _should_ above to save existing libraries out there, we I guess we should also say _should_ here to save existing tools?
```suggestion
Vendors should use a \lstinline[language=CSS]!$NameOfVendor$-! prefix to style vendor generated HTML content surrounding the user provided documentation.
```